Tests/SwiftGenKitTests/TestsHelper.swift

```diff
@@ -76,16 +76,16 @@ func XCTDiffContexts(_ result: [String: Any],
                      sub directory: Fixtures.Directory,
                      file: StaticString = #file,
                      line: UInt = #line) {
-  #if GENERATE_CONTEXTS
+  if ProcessInfo().environment["GENERATE_CONTEXTS"] == "YES" {
```

Comment:
If I read the `xcscheme` properly, you gave it the value `Test` not the value `YES`, so you should check against that `Test` value not against `YES`?